Client/Frontend/Home/HomePanelViewController.swift

```diff
@@ -9,7 +9,7 @@ import UIKit
 private let BackgroundColor = UIColor(red: 45.0 / 255, green: 52.0 / 255, blue: 66.0 / 255, alpha: 1)
 private let NormalIconColor = UIColor.lightGrayColor()
 private let SelectedIconColor = UIColor(red: 62.0 / 255, green: 136.0 / 255, blue: 255.0 / 255, alpha: 1)
-private let ContainerHeight = 40
+let ContainerHeight = 40
```

Comment:
Since this is now a global var, it should be named `HomePanelContainerHeight` to make its purpose clear and prevent namespace collisions.